src/nucleic_acid/nucleic_acid.jl

```diff
@@ -5,26 +5,16 @@ const _πT(mod) = _πACGT(mod)
 const _πACGU(mod) = _πACGT(mod)
 
 
-@inline function _πACGT(mod::NASM)
-  return 0.25
-end
-
-@inline function _π(mod::NASM)
-  return SVector(_πA(mod), _πC(mod), _πG(mod), _πT(mod))
-end
-
+_πACGT(mod::NASM) = 0.25
 
-@inline function _πR(mod::NASM)
-  return _πA(mod) + _πG(mod)
-end
+_π(mod::NASM) = SVector(_πA(mod), _πC(mod), _πG(mod), _πT(mod))
 
+_πR(mod::NASM) = _πA(mod) + _πG(mod)
 
-@inline function _πY(mod::NASM)
-  return _πT(mod) + _πC(mod)
-end
+_πY(mod::NASM) = _πT(mod) + _πC(mod)
 
 
-doc"""
```

Comment:
The doc prefix has been necessary to have LaTeX properly generate with documenter.jl - unless I'm missing something.

Reply:
Ah! I missed that doc is required for LaTeX. Ok I will change back.

Reply:
This is changed back now :+1: